easybuild/easyblocks/b/boost.py

```diff
@@ -89,6 +89,8 @@ def patch_step(self):
     def configure_step(self):
         """Configure Boost build using custom tools"""
 
+        configopts = "--prefix=%s" % self.objdir
```

Comment:
Are you sure I want/can use `.update()` on `self.cfg['configopts']`? EB complains on not being able to call `.update()` on a string.